### PR TITLE
New Module: Cloud Volume for azure NetApp Volume

### PR DESCRIPTION
**Pull request by carchi8py**
##### SUMMARY
Cloud Volume for azure NetApp Volume

##### ISSUE TYPE

- New Module Pull Request

##### COMPONENT NAME
- azure_rm_netapp_volume.py 

##### ADDITIONAL INFORMATION

```paste below

```

### CODE REVIEW

**Comment by Fred-sun** [comment]
@carchi8py Thank you for contributor this PR. Could you fix the CI fail, when you finished changed, I will push for review! Thanks a lot!

**Comment by carchi8py** [comment]
@fred-sun the fix for this is in pr #59939, i'm talking with Matt and the Redhat team on how to handle the unit test. We might just drop them.

**Comment by Fred-sun** [comment]
@carchi8py How about the PR going ahead? Any update? Thanks a lot!

**Comment by anshulbehl** [comment]
@carchi8py This is still using get_client_from_cli_profile, Please make the changes in azure_rm_common.py and commit a new change.

File: lib/ansible/modules/cloud/azure/azure_rm_netapp_volume.py [hunk 1]
```diff
@@ -0,0 +1,251 @@
...
+from ansible.module_utils.netapp_module import NetAppModule
+from ansible.module_utils.basic import AnsibleModule
+import traceback
+
+AZURE_OBJECT_CLASS = 'NetAppAccount'
+HAS_AZURE_MGMT_NETAPP = False
+try:
+    from azure.mgmt.netapp.azure_net_app_files_management_client import AzureNetAppFilesManagementClient
+    from azure.mgmt.netapp.models import Volume
+    from azure.common.client_factory import get_client_from_cli_profile
```
Comment (anshulbehl) [inline lib/ansible/modules/cloud/azure/azure_rm_netapp_volume.py:129]: Please change this like, https://github.com/ansible/ansible/pull/59939

**Review by anshulbehl — CHANGES_REQUESTED** [review]

**Comment by Fred-sun** [comment]
kindly ping

**Comment by Fred-sun** [comment]
@carchi8py Are you still there? Can you update PR according to comments? Thank you very much!

File: lib/ansible/modules/cloud/azure/azure_rm_netapp_volume.py [hunk 2]
```diff
@@ -0,0 +1,251 @@
...
+                    'supported_by': 'certified'}
+
+
+DOCUMENTATION = '''
+---
+module: azure_rm_netapp_volume
+
+short_description: Manage NetApp Azure Files Volume
+version_added: "2.9"
+author: NetApp Ansible Team (@carchi8py) <ng-ansibleteam@netapp.com>
```
Comment (Fred-sun) [inline lib/ansible/modules/cloud/azure/azure_rm_netapp_volume.py:21]: You should move the author to the end of the document and change the format as follows:
```suggestion
author:
    - NetApp Ansible Team (@carchi8py)
```

File: lib/ansible/modules/cloud/azure/azure_rm_netapp_volume.py [hunk 3]
```diff
@@ -0,0 +1,251 @@
...
+    state:
+        description:
+            - State C(present) will check that the volume exists with the requested configuration.
+            - State C(absent) will delete the volume.
+        default: present
+        choices:
+            - absent
+            - present
+
+'''
```
Comment (Fred-sun) [inline lib/ansible/modules/cloud/azure/azure_rm_netapp_volume.py:82]: Add the return value and example.

**Comment by Fred-sun** [comment]
@carchi8py Another, you should add document for test_azure_rm_netapp_volume. Thank you very much!

**Comment by carchi8py** [comment]
Hey @Fred-sun since we were told there would be no new modules accepted in ansible after 2.9 (and this missed that window) we made this in to a collection here. 

https://github.com/ansible/ansible_collections_netapp/tree/master/ansible_collections/netapp/azure

**Comment by Fred-sun** [comment]
@carchi8py I see, Thanks for your info. If we are not adding to ansible 2.9, maybe we need to turn off this PR. Thank you very much!

**Comment by Fred-sun** [comment]
@mybayern1974 If we can't add a new Module to Ansible after 2.9, what should we do with this kind of PRs? Please check! Thank you very much!

**Comment by Fred-sun** [comment]
@carchi8py Would you please help fixes the CI fail when you're free？It will push for PR review and merge. Thank you very much!

**Comment by Fred-sun** [comment]
@carchi8py Thank you for taking the time to contribute to this PR. We will transfer ansible 's azure module related Issue and PR to azure collection (https://github.com/ansible-collections/azure/pulls), can you transfer the Issue to azure collection repo?

**Comment by Akasurde** [comment]
Hi @carchi8py, Thank you very much for your interest in Ansible. This plugin/module is no longer maintained in this repository and has been migrated to https://github.com/ansible-collections/azure<br><br>If you have further questions please stop by IRC or the mailing list:<br><br>* IRC: #ansible on irc.freenode.net<br>* mailing list: https://groups.google.com/forum/#!forum/ansible-project<br><br>needs_info<br>

**Pull request closed without merging**